content/en/docs/overview/singleton-sdk/Angular Client Introduction/i18n-service.md

```diff
@@ -0,0 +1,144 @@
+---
+title: "I18n Service"
```

Comment:
It displays  extremely ugly on UI as below.
![image](https://user-images.githubusercontent.com/8565741/78869987-01052500-7a78-11ea-8667-4a834a44e94e.png)

Reply:
I have fixed it.